### ISSUE TITLE
Chinese translation update

### ISSUE THREAD

**Issue by Ventulus-lab** [Truinto/ONI-Modloader-SimpleMods#27]
[strings_zh.zip](https://github.com/Truinto/ONI-Modloader-SimpleMods/files/7845608/strings_zh.zip)
I update some strings translation more accurately after testing

**Comment by Ventulus-lab** [comment]
oh,no!
I find it doesn't work for skill station.
though I have translated the strings of Atrributes & Traits ..etc, but in game the entry of skill station shows English and Missing Strings.
Take a look please

**Comment by Truinto** [comment]
Oh weird. I am sure I checked that these entries are translatable. Something must have changed that it's not working anymore. I will need time to investigate.

**Comment by Truinto** [comment]
Made progress. Still missing the category headers...

![pic](https://user-images.githubusercontent.com/47917181/149586523-5ef2877d-a67b-4c55-ba81-d4180685818b.png)

**Comment by Truinto** [comment]
Ah, I just realized why. The IDs were missing from the *.pot table before my latest patches. So naturally nobody could have translated them :)
Would you help me with this. Here is the new pot file. Also the next mod version in case you want to try it before release.

[strings_.zip](https://github.com/Truinto/ONI-Modloader-SimpleMods/files/7873286/strings_.zip)
[CustomizeBuildingsV57pre.zip](https://github.com/Truinto/ONI-Modloader-SimpleMods/files/7873290/CustomizeBuildingsV57pre.zip)
.

**Comment by Ventulus-lab** [comment]
well,I was confused by your mod AirConditionerSliders.
Can you explain these parameters and formulas?
about AirConditionerAbsolutePowerFactor   factorDPU in your mod 
and WattsNeededWhenActive BaseWattageRating  in game

**Comment by Ventulus-lab** [comment]
done
[strings_zh.pot.zip](https://github.com/Truinto/ONI-Modloader-SimpleMods/files/7873810/strings_zh.pot.zip)

**Comment by Truinto** [comment]
Let's see. You have
- temperature slide: let's you choose the target temperature; everything going through has that exact temperature
- DPU slider: let's you choose the heat transfer rate; this defines how quickly the building heats up. Say you put through 10kg of super-coolant and cool it by 100°C, that would melt the building in seconds. This setting will instead reduce the package size, for example to 100 gram.

factorDPU isn't in the config file. I assume you looked at the source code. I made it more complicated than it had to. It runs faster, but even if you fill the map with air conditioners you wouldn't notice a difference :P

If I remember correctly it works like this: Take the buildings normal power consumption, multiply it with the factor of maximum allowed DPU divide through actual DPU. Since that usually ends up at a very low value I added 100W as a base cost. AirConditionerAbsolutePowerFactor will multiply the cost, so it scales faster to the maximum. It never costs more than the buildings normal power consumption (1200W or 240W, I think).

BaseWattageRating is used by the electric net to determine how much power is actually consumed
WattsNeededWhenActive is the normal power consumption without mod (1200W or 240W)
DPU is the heat unit
